bodyprocessors/xml.go

```diff
@@ -32,10 +32,15 @@ func (*xmlBodyProcessor) ProcessResponse(reader io.Reader, v rules.TransactionVa
 	return nil
 }
 
-func readXML(reader io.Reader) ([]string, []string, error) {
+func readXML(reader io.Reader, strict bool) ([]string, []string, error) {
```

Comment:
How would this value be set to true?

Reply:
it's for testing purposes, that's the only way we can write negative and positive tests

Reply:
It's a test for logic that can never be executed in practice, I think normally we wouldn't add tests for that since the behavior of the logic gets less clear